hal/src/generate/FRCNetComm.java.in

```diff
@@ -5,7 +5,10 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-// Autogenerated by wpilibj_frcnetcomm.py. Do not manually edit this file.
+/*
+ * Autogenerated file! Do not manually edit this file. This file is regenerated
```

Comment:
What the comment says.

Reply:
Just making the style of the comment match WPILibVersion.java's:

https://github.com/wpilibsuite/allwpilib/blob/3eae079db4788c260cada19cb2e150d809d91abb/wpilibj/src/generate/WPILibVersion.java.in#L3-L6

Reply:
Oh.  I suppose this will make travis fail... 🤦‍♂️

Reply:
In my opinion, we should change WPILibVersion.java.in to use the shorter comment since it conveys the same information.

Reply:
I couldn't care less which one is used, so long as it's generated by the script, and not manually edited :P